web/src/title/loader.ts

```diff
@@ -19,8 +20,12 @@ import {
 } from "../stores";
 import { Backend } from "../wasm";
 
+export let loadingMessage = writable("");
+export let loadingProgress: Writable<number | null> = writable(null);
+
 /**
- * Loads an existing project by its ID
+ * Loads an existing project by its ID. Changes loadingMessage and
```

Comment:
Updating these stores was the most sane way I could think of to plumb updates out